### PR TITLE
[RO] Add `Beatmapping` and `Installation and registration`

### PR DESCRIPTION
**Pull request by 28edx**
Translation for the `Beatmapping` and `Installation and registration` articles

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Beatmapping/ro.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+## Cum încarc/actualizez beatmap-ul meu? {id=upload-beatmap}
+
+**Poți face asta prin editorul din joc.**
+
+Pentru a îți încărca beatmap-ul, du-te în editor și apasă pe `Fișier` > `Uploadează Beatmap`.
+
+Asta îți va face beatmap-ul disponibil pentru a putea fi descărcat de alții de pe site. Actualizarea unui beatmap are același proces pentru un beatmap deja încărcat.
+
+Fiecare utilizator are un număr limitat de beatmap-uri pe care le poate încărca simultan. Lăsarea unui beatmap inactiv timp de o lună (28 de zile), prin neactualizarea ei, o va trimite în [Beatmap-uri inactive](/wiki/Beatmap/Category#graveyard). Pentru a readuce un beatmap din inactivitate înapoi în statusul de În așteptare sau În lucru, actualizeaz-o.
```
Comment (NiceAesth) [inline wiki/Help_centre/Beatmapping/ro.md:29]:
```suggestion
Fiecare utilizator are un număr limitat de beatmap-uri pe care le poate încărca simultan. Lăsarea unui beatmap inactiv timp de o lună (28 de zile), prin neactualizarea lui, îl va trimite în [Beatmap-uri inactive](/wiki/Beatmap/Category#graveyard). Pentru a readuce un beatmap din inactivitate înapoi în statusul de În așteptare sau În lucru, actualizează-l.
```

File: wiki/Help_centre/Beatmapping/ro.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Editorul îmi spune că „Beatmap-ul pe care încercați să îl uploadați nu este al dumneavoastră!”. {id=wrong-author}
+
+**Asta se poate întâmpla dacă creatorul beatmap-ului nu este setat corespunzător în fișierele beatmap-ului.**
+
+În editor, fă clic în meniu pe `Fișier` , apoi `Deschide .osu în Notepad`. Verifică linia `Creator:`, sub secțiunea `Metadata`, și asigură-te că este setată exact cu numele tău de utilizator inclusiv majusculele.
+
+Dacă nu este, modifică manual asta pentru fiecare dificultate a beatmap-ului tău, reîncarcă beatmap-ul și ar trebui să o poți încărca.
```
Comment (NiceAesth) [inline wiki/Help_centre/Beatmapping/ro.md:37]:
```suggestion
Dacă nu este, modifică manual asta pentru fiecare dificultate a beatmap-ului tău, după care reîncarcă beatmap-ul și ar trebui să funcționeze.
```

File: wiki/Help_centre/Beatmapping/ro.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Dacă încă întâmpini probleme sau vrei mai multe informații despre asta, citește [pagina de probleme BSS](/wiki/Guides/BSS_issues).
```
Comment (NiceAesth) [inline wiki/Help_centre/Beatmapping/ro.md:39]:
```suggestion
Dacă încă întâmpini alte probleme sau dorești mai multe informații, citește [pagina de probleme BSS](/wiki/Guides/BSS_issues).
```

File: wiki/Help_centre/Beatmapping/ro.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Sunt un suporter osu!, dar nu am 20 de sloturi în așteptare! {id=upload-slots}
+
+**Numărul sloturilor disponibile depinde de numărul beatmap-urilor clasate pe care le ai. Pentru mai multe detalii, citește [suporter osu! § limitele crescute](/wiki/osu!supporter#increased-limits).**
+
+## Beatmap-ul/conținutul meu a fost reîncărcat de către alt utilizator fără consințământul meu, și vreau ca copia să fie dată jos! {id=report-content-dmca}
```
Comment (NiceAesth) [inline wiki/Help_centre/Beatmapping/ro.md:45]:
```suggestion
## Beatmap-ul/conținutul meu a fost reîncărcat de către alt utilizator fără consințământul meu și doresc să fie dată jos copia! {id=report-content-dmca}
```

File: wiki/Help_centre/Beatmapping/ro.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+**Te rugăm să depui o reclamație DMCA la [copyright@ppy.sh](mailto:copyright@ppy.sh) în conformitate cu [politica drepturilor de autor osu!](/wiki/Legal/Copyright).**
+
+Orice fel de conținut încărcat fără permisiune poate fi considerat ca o încălcare a drepturilor de autor dacă proprietarul original dorește asta. Solicitarea eliminării necesită o procedură legală pentru a asigura o procedură corectă.
```
Comment (NiceAesth) [inline wiki/Help_centre/Beatmapping/ro.md:49]:
```suggestion
Orice fel de conținut încărcat fără permisiune poate fi considerat o încălcare a drepturilor de autor dacă proprietarul original dorește asta. Solicitarea eliminării necesită o procedură legală pentru a asigura o procedură corectă.
```

File: wiki/Help_centre/Beatmapping/ro.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+Te rugăm să **nu** depui o reclamare folosind fie [comanda din joc `!report`](/wiki/Reporting_bad_behaviour#in-game-chat) sau [site-ul osu!](/wiki/Reporting_bad_behaviour#website), deoarece membrii [NAT](/wiki/People/Nomination_Assessment_Team) și [GMT](/wiki/People/Global_Moderation_Team) care gestionează aceste rapoarte nu sunt în măsură să se ocupe de probleme juridice.
```
Comment (NiceAesth) [inline wiki/Help_centre/Beatmapping/ro.md:51]:
```suggestion
Te rugăm să **nu** depui o reclamare folosind fie [comanda din joc `!report`](/wiki/Reporting_bad_behaviour#in-game-chat) sau [site-ul osu!](/wiki/Reporting_bad_behaviour#website), deoarece membrii [NAT](/wiki/People/Nomination_Assessment_Team) și [GMT](/wiki/People/Global_Moderation_Team) care gestionează aceste sesizări nu sunt în măsură să se ocupe de probleme juridice.
```

**Review by NiceAesth — CHANGES_REQUESTED** [review]

**Comment by NiceAesth** [comment]
Suggestions can be applied wholesale

**Review by NiceAesth — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**